packages/bootstrap-vue-next/src/components/BNavbar/BNavbar.vue

```diff
@@ -9,19 +9,19 @@
 
 <script setup lang="ts">
 import {computed, provide, toRef} from 'vue'
-import type {Booleanish, Breakpoint, ColorVariant} from '../../types'
-import {useBooleanish, useContainerClasses} from '../../composables'
+import type {Breakpoint, ColorVariant} from '../../types'
+import {useContainerClasses} from '../../composables'
 import {navbarInjectionKey} from '../../utils'
 
 const props = withDefaults(
   defineProps<{
-    autoClose?: Booleanish
-    container?: 'fluid' | Booleanish | Breakpoint
+    autoClose?: boolean
+    container?: 'fluid' | boolean | Breakpoint
```

Comment:
```suggestion
    container?: boolean | 'fluid' | Breakpoint
```

(Fixes a preexisting bug)

If you find any prop definitions that are similar to this, you want to make sure `boolean` comes first. Read https://vuejs.org/guide/components/props.html#boolean-casting 

If you don't mind, please take a browse through the props definitions and make sure boolean comes first